sqrl-tools/sqrl-packager/src/main/java/com/datasqrl/packager/preprocess/JarPreprocessor.java

```diff
@@ -89,8 +89,24 @@ private Void processJarEntry(JarEntry entry, JarFile file,
       processorContext.addLibrary(path);
     }
 
+    copyFileWithStructure(path, processorContext.getBuildDir());
```

Comment:
processorContext.addDependency(path); to copy files

Reply:
oh wow. cool!